.vscode/settings.json

```diff
@@ -0,0 +1,20 @@
+{
+    "workbench.colorCustomizations": {
```

Comment:
Please don't force your colorCustomizations on other developers. You can keep these in user settings.

Reply:
✔️ - added .vscode to the general .gitignore in case anyone else uses the peacock extension (forces color stylings)